routers/posts.py

```
@@ -1,29 +1,278 @@
+    try:
+        postId_int = int(postId)
+    except ValueError:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail={
+                "status": "error",
+                "data": {
+                    "message": "존재하지 않는 게시글입니다."
+                }
+            }
+        )
```

Ruff (0.14.13)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)